app/livechat/client/stylesheets/omnichannel.less

```diff
@@ -243,52 +243,47 @@
 	flex-direction: column;
 }
 
-.lc-chart-options {
+.omnichannel-chart-options {
 	// flex: 1;
 }
 
 .border-component-td {
   padding: 0.5rem;
 }
 
-.lc-analytics-chart-container {
+.omnichannel-analytics-chart-container {
 	// flex-grow: 100%;
 	flex: auto;
 	position: relative;
 	// height: inherit;
 }
 
-.lc-analytics-header {
+.omnichannel-analytics-header {
 	float: right;
-  & > .lc-date-picker-btn {
-    font-weight: inherit;
-    font-size: inherit;
-    color: #383838;
-    //line-height: 1;
-    padding-top: 23px;
-  }
 
-  & > .lc-date-picker-btn {
-    // padding-top: 15px;
-    // padding-right: 25px;
-    // padding-left: 10px;
-    position: relative;
+	& > .omnichannel-date-picker-btn {
+		font-weight: inherit;
+		font-size: inherit;
+		padding-top: 23px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "padding-top" to come before "color" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4828896)